docs/Authenticating-Users-via-keycloak-SAML.md

```diff
@@ -107,16 +111,22 @@ keytool -genkey -alias secure-key -keyalg RSA -keystore samlKeystore.jks
     saml.logout.url=/
 ```
 
-## Integrating Keycloak with your company-wide authentication service
+## Integrate Keycloak with your company-wide authentication service
 
-When integrating Keycloak with your company-wide authentication service, the user credentials will reside at your institute's users directory and Keycloak
-will work as a proxy between your LDAP (or SAML) service and cBioPortal.
+When integrating Keycloak with your company-wide authentication service, the user credentials will reside at your institute's users directory and Keycloak will work as a proxy between your LDAP (or SAML) service and cBioPortal.
 
-Please refer to Keycloak documentation [http://www.keycloak.org/documentation.html](http://www.keycloak.org/documentation.html) for more information on how to integrate Keycloak with your local LDAP/SAML service.
+Please refer to [the Keycloak documentation](https://keycloak.gitbooks.io/documentation/server_admin/topics/user-federation.html) for more information on how to integrate Keycloak with your local LDAP/SAML service.
 
 ## Authorization with Keycloak
```

Comment:
No, a level 1 heading is semantically the heading for the page as a whole, and displayed as such in the Readthedocs outline. The top-level sections within the page are level 2 headings.